pkg/jsx/jsx-runtime.js

```diff
@@ -0,0 +1,16 @@
+import { h, text } from 'hyperapp'
+
+const jsx = (type, { children = [], ...props }) =>
+  typeof type === 'function'
+    ? type(props || [], children)
+    : h(
+        type,
+        props || {},
+        []
+          .concat(children)
+          .map((any) =>
+            typeof any === 'string' || typeof any === 'number' ? text(any) : any
+          )
+      )
+
+export { jsx, jsx as jsxs, jsx as jsxDEV }
```

Comment:
What is `jsxs`?

Reply:
So from my understanding, the new transform will use the `jsx` export for single elements with no children, `jsxs` when the element has children (I think this is so we can avoid doing the concat and the "default to array" or conditional logic) and `jsxDEV` so we can add development utilities. Because the function in the PR handles both cases (single and with children), we can just use the same function exported with the 2 names. [Same thing as preact does](https://github.com/preactjs/preact/blob/master/jsx-runtime/src/index.js#L67-L72).

Reply:
Turns out `jsx` is for 0 or 1 child, `jsxs` is for many children. Updated the PR to take advantage of it (eliminated the `.concat` which should help perf)

Reply:
Is there any documentation of `jsxs`?

Reply:
Best I could find:
https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#react-automatic-runtime
(second example block shows with jsxs)

Also: https://github.com/preactjs/preact/blob/master/jsx-runtime/src/index.js#L7-L10

And: https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform